.github/actions/compute-sha256/compute-sha256.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [[ ! -f "$UNTRUSTED_PATH" ]]; then
+    echo "File $UNTRUSTED_PATH not present"
+    exit 5
+fi
+
+DIGEST=$(sha256sum "$UNTRUSTED_PATH" | awk '{print $1}')
+
+echo "::set-output name=sha256::$DIGEST"
```

Comment:
nit: while we're still using bash scripts, let's try to get in the habit of using lowercase for local unexported variables so they can be more easily differentiated.

```suggestion
digest=$(sha256sum "$UNTRUSTED_PATH" | awk '{print $1}')

echo "::set-output name=sha256::$digest"
```